force-app/main/default/classes/Collection Recipes/AccountShippingCountryComparator.cls

```diff
@@ -0,0 +1,17 @@
+/**
+ * @description An example implementation of Comparator Interface
+ * In this example we show how to sort all the accounts by their country names in alphabetical order
+ **/
+
+public with sharing class AccountShippingCountryComparator implements Comparator<Account> {
```

Comment:
Just a quick thought: we use two basic custom comparators in our recipes. Could we make this one a bit more interesting by adding a sort order parameter (ascending/descending) to the constructor?
We don't need to demo both sorting order but mention that this is a possibility.

Reply:
I agree @pozil! Will enhance this one a bit to show sort order by asc and desc

Reply:
Enhanced the code now to use a sort order.